### ISSUE TITLE
Incognito Snapshots are Enabled by default

### ISSUE THREAD

**Issue by JustAnUserLol** [bromite/bromite#1816]
### Preliminary checklist

- [X] I have read the [README](https://github.com/bromite/bromite/blob/master/README.md).
- [X] I have searched the existing issues for my problem. This is a new ticket, NOT a duplicate or related to another open issue.
- [X] I have updated Bromite to the latest version. The bug is reproducible on this latest version.

### Can the bug be reproduced with corresponding Chromium version?

No

### Bromite version

97.0.4692.106

### Device architecture

armeabi-v7a

### Android version

8.1

### Device model

`Samsung Galaxy J2 Core`

### Changed flags

disable-webgl, disable-pull-to-refresh-effect, offline-indicator-v2, enable-vibration, cleartext-permitted

### Is this bug about the SystemWebView?

No

### Is this bug happening in an incognito tab?

Yes

### Is this bug caused by the adblocker?

No

### Is this bug a crash?

No.

### Describe the bug

Incognito Snapshots are Enabled by default.

### Steps to reproduce the bug

Go to google.com
See Snapshot in android recents

### Expected behavior

it should not show snapshots in Incognito Mode

### Screenshots

![image](https://user-images.githubusercontent.com/95953932/154864223-3ad077db-7af8-40f1-bfe3-e9b691a50426.jpg)

**Comment by csagan5** [comment]
They are not enabled by default; I cannot reproduce this issue. Is anyone else capable of reproducing this?

**Comment by taivlam** [comment]
Yes, I can reproduce this.  I'm observing this on my Samsung Galaxy Tab S6 Lite  (`gta4xlwifi`) with the following:

* Bromite version: 98.0.4758.108
* OS: LineageOS 18.1 for microG /Android 11
    * February 18, 2022 update

I also observe this on my Pixel 4a (`sunfish`) with the latest GrapheneOS version (build number SQ1A.220205.002.2022021721).

I have Incognito mode enable by default, which should disable screenshots via "Settings" (the 3 vertical dots menu) > "Privacy and security" > "Open links in incognito tabs always".  However, I can always see what's on my screen, even in the Recents app overview.

**Comment by uazo** [comment]
I confirm, it happens to me too. the default value (disabled) of the flag appears to be ineffective, I think there has been a change from upstream

**Comment by taivlam** [comment]
Also screenshots aren't being blocked on my Pixel 5a (`barbet`) with GrahpeneOS (build number SQ1A.220205.002.2022021721) in Bromite.

For reference (not sure if this is relevant), Vanadium in Incognito mode successfully blocks screenshots in GrapheneOS on both Pixel 4a and Pixel 5a.

The only change in Android 12/Android S for apps that block screenshots from a user's perspective is: the app will still be "active" when navigating from the app directly to the Recents overview.  However, after navigating elsewhere (such as the Home screen or another full screen app) will subsequently activate screenshot blocking in Recents.

**Comment by csagan5** [comment]
Enabled by default since v97 by mistake (@uazo, see https://github.com/uazo/bromite/blob/master-v97/build/patches/Bromite-auto-updater.patch#L1329).

Will be fixed on next release.

**Comment by uazo** [comment]
ops... sorry

**Comment by taivlam** [comment]
I just updated to the latest release and the screenshot blocking now works again on my Galaxy Tab S6 Lite and Pixel 4a.

**Comment by csagan5** [comment]
Fixed in `99.0.4844.55`.
